magentoerpconnect/CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 Changelog
 ---------
 
+3.0.0 (unreleased)
```

Comment:
As I do not know when it will be released, I will make a commit to change this date once merged.
Or, I can put today if you assure me it will be merged today ;-)